complete/03_ingress/app.ingress.yaml

```
@@ -0,0 +1,23 @@
+# You may need to change the ingress class below,
+# to match the Nginx Ingress configuration.
+#
+# Access this INgress by specifying the HOST header.
+# Replace ${load_balancer_ip} with the IP of an Ingress Controller LB.
+# curl --header 'HOST: test.domain.com' ${load_balancer_ip}/
+---
+apiVersion: networking.k8s.io/v1beta1
```

networking.k8s.io/v1beta1 is being deprecated in 1.19 so I believe this newer API version should be used if you're using a semi recent version of kube in the workshop.

```suggestion
apiVersion: networking.k8s.io/v1
```